Readme.md

```diff
@@ -246,6 +246,7 @@ TIPS
  - [Distributed parallel test (e.g. Travis Support)](https://github.com/grosser/parallel_tests/wiki/Distributed-Parallel-Tests-and-Travis-Support)
  - Debug errors that only happen with multiple files using `--verbose` and [cleanser](https://github.com/grosser/cleanser)
  - Shell alias: `alias prspec='parallel_rspec -m 2 --'`
+ - [Spring] to use spring you have to [patch it](https://github.com/grosser/parallel_tests/wiki) 
```

Comment:
Another suggestion: Instead of maintaining the code on the wiki, put it in the code base, and `require "parallel_tests/spring_config" if defined?(::Spring)`